cli/config.go

```diff
@@ -78,6 +78,10 @@ type Config struct {
 		APIURL string `env:"API_URL"`
 		APIKey string `env:"API_KEY"`
 	} `envPrefix:"OAI_"`
+	Projects struct {
+		APIURL string `env:"API_URL"`
+		APIKEY string `env:"API_KEY"`
```

Comment:
better use `APIKey` for consistency (see lines above)

Reply:
Done.